modules/ocf_lb/manifests/init.pp

```diff
@@ -0,0 +1,21 @@
+class ocf_lb (
+  Array $vip_names,
+  String $keepalived_secret_lookup,
+) {
+  $virtual_addresses = $vip_names.map |$vip| {
+    ldap_attr($vip, 'ipHostNumber')
+  }
+  $virtual_addresses_v6 = $vip_names.map |$vip| {
+    regsubst(ldap_attr($vip, 'ipHostNumber'), '^(\d+)\.(\d+)\.(\d+)\.(\d+)$', '2607:f140:8801::1:\4')
```

Comment:
sigh... I guess this is fine

Reply:
copy pasted from https://github.com/ocf/puppet/blob/2623cd0c204aec6f1d0496575c6cb06b5ffdda5c/modules/ocf/manifests/networking.pp#L9 😬